style.css

```diff
@@ -5,99 +5,258 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
-  font-family: "Poppins", sans-serif;
 }
 
+.html,
 body {
-  justify-content: center;
-  align-items: center;
-  background-image: url("background.png");
-  background-repeat: no-repeat;
-  background-position-y: 90%;
-  width: 375px;
-  height: 718px;
+  font-family: Poppins, sans-serif;
+}
+
+a:hover {
+  color: #172b4d;
 }
 
-/* LOGO AND NAVBAR STYLING  */
-.head {
+.blue {
   color: #6070ff;
-  text-align: center;
 }
 
-header {
+/* Css for Header */
+.header_nav {
   display: flex;
-  padding: 24px;
+  min-height: 10vh;
+  padding: 10px 20px;
+  align-items: center;
   justify-content: space-between;
 }
 
-.logo {
-  margin-top: 14px;
-  font-size: 18px;
+.header_logo {
+  font-size: 22px;
   font-weight: 700;
   line-height: 20px;
-  width: 76px;
-  height: 20px;
 }
 
-.logo a {
+.header_logo a {
   text-decoration: none;
-  cursor: pointer;
 }
 
-header.navbar i {
-  font-size: 22px;
-  padding-right: 27px;
+.header_menu {
+  padding: 3px 5px;
+}
+
+.menu_icon {
+  width: 24px;
+  height: auto;
 }
 
-/* TEXT STYLING  */
-.text {
-  padding: 0 24px;
+/* Css styles for main_container */
+.main_container {
+  display: grid;
+  row-gap: 25px;
 }
 
-.text h1 {
+/* Css Styles for headline section */
+.headline {
+  padding: 25px;
   display: flex;
-  align-items: center;
+  background-size: contain;
+  flex-direction: column;
+  background-repeat: space;
+  background-image: url("./images/header-shapes-mobile.png");
+}
+
+.headline_title {
   font-size: 40px;
+  margin-top: 60px;
+  margin-bottom: 20px;
   font-weight: 700;
-  line-height: 52px;
   color: #172b4d;
-  margin-bottom: 28px;
-  margin-top: 124px;
-  left: 24px;
-  width: 307px;
-  height: 150px;
-  padding-top: 24px;
+  line-height: 52px;
 }
 
-.text p {
+.headline_content {
+  margin: 15px 0;
   color: #344563;
   font-weight: 400;
-  font-size: 16px;
-  line-height: 24px;
-  order: 1;
-  margin-bottom: 28px;
-  margin-top: 15px;
-  top: 276px;
-  left: 24px;
-  width: 307px;
-  height: 146px;
+  background: #fff;
 }
 
-.text .social-text h5 {
+.social_text {
   font-size: 16px;
-  font-weight: 500;
-  color: #7f8cff;
-  line-height: 24px;
-  margin-top: 15px;
-  height: 24px;
+  font-weight: normal;
+}
+
+.social_links {
+  display: flex;
+  list-style: none;
+  margin-top: 10px;
+}
+
+.social_link a {
+  width: 15px;
+  margin-top: 10px;
+  margin-right: 10px;
+}
+
+.social_link a:hover {
+  color: #172b4d;
+}
+
+.social_icon {
+  max-width: 100%;
+}
+
+/* Css Styles for work section */
+.work {
+  margin: 25px 0;
+  display: grid;
+  row-gap: 25px;
+  padding: 10px;
+  background: #e5e5e5;
+  justify-items: center;
+}
+
+.work_card {
+  background: #fff;
+  border-radius: 10px;
+  min-height: 3em;
+  border: 1px solid #c6c6c6;
+  padding: 20px 15px;
+  display: grid;
+}
```

Comment:
- [ ] Your work card design is great and very similar to the Figma template, but the project requirement specified that you should combine Grid and Flexbox to style the works section, and the work card should be styled with Flexbox. Please change the `display` attribute to `flex` to match with the project requirements.